index.html

```diff
@@ -44,7 +44,8 @@ <h1 class="text-white text-2xl" style="text-shadow: 0 0 10px red;"></h1>
                         <input id="rating" type="text" value="displaySearch" class="border-2 w-auto mt-2 rounded-md justify-items-end ">
                         <label for="releaseDate" class="block text-lg text-white mb-10px ml-2 justify-items-end">ReleaseDate</label>
                         <input id="releaseDate" type="text" value="displaySearch" class="border-2 w-auto mt-2 p-10px rounded-md justify-items-end">-->
-                    <div>
+                    <ul>
```

Comment:
we don't need  a fun fact title over the api info displayed

Reply:
OK!